### PR TITLE
Website | Gallery: Donut Example #38

### PR DESCRIPTION
**Pull request by lee00678**
Add a gallery example which showcases the use of a basic donut chart. ([issue](https://github.com/unovis/issues-only/issues/38))

- [x] Create a folder in `shared/examples` directory called basic-donut-chart with all the relevant files
- [x] Find dataset and create example in `basic-donut-chart.tsx` 

Add code for other frameworks: 
- [x] TypeScript
- [x] Angular
- [x] Svelte
- [x] Vue

And then add screenshots to `_shared/previews`:

- [x] basic-donut-chart.png
- [x] basic-donut-chart-dark.png

### CODE REVIEW

**Comment by rokotyan** [comment]
@reb-dev It might be a good time to enforce the Commit Lint formats: https://github.com/f5/unovis/pull/300

**Comment by lee00678** [comment]
> Great job @lee00678!
> 
> I'm a little concerned about the visual aspects of the new example because it's inconsistent with other examples and our Unovis brand. Specifically, the colors and the capitalized text in the legend. 

I'm happy to make any changes! Also nice to "meet" both of you @rokotyan and @Dovlash. Yes, I used D3's color theme as I ran out of the default colors for donut chart. I can probably look into how to get unovis' color theme rendered, and I will update the legend as well.

**Comment by rokotyan** [comment]
Nice to meet you too @lee00678!

I see, I think instead of showing so many slices, we can merge smaller ones into a big "Others" one.
Here's two great posts about Pie Charts and Colors in data vis:
- [What to consider when creating pie charts](https://blog.datawrapper.de/pie-charts/)
- [10 ways to use fewer colors in your data visualizations](https://blog.datawrapper.de/10-ways-to-use-fewer-colors-in-your-data-visualizations/)

**Comment by lee00678** [comment]
> Nice to meet you too @lee00678!
> 
> I see, I think instead of showing so many slices, we can merge smaller ones into a big "Others" one. Here are two great posts about Pie Charts and Colors in data vis:
> 
> * [What to consider when creating pie charts](https://blog.datawrapper.de/pie-charts/)
> * [10 ways to use fewer colors in your data visualizations](https://blog.datawrapper.de/10-ways-to-use-fewer-colors-in-your-data-visualizations/)

I like the idea of combining the smaller categories. Made an update on that.

**Comment by rokotyan** [comment]
Nice, thanks for the update!

**Comment by rokotyan** [comment]
@lee00678 Btw, if you use Discord, we've a space there which we're planning to make public soon. Feel free to join: https://discord.gg/zATXusWgCP

File: packages/shared/examples/examples-list.tsx [hunk 1]
```diff
@@ -25,6 +25,13 @@ export const examples: ExampleCollection[] = [
       require('./horizontal-stacked-bar-chart').default,
     ],
   },
+  {
+    title: 'Donut Charts',
```
Comment (reb-dev) [inline packages/shared/examples/examples-list.tsx:29]: Instead of making a new category, I would add it as the first entry to "Circular Charts"

**Review by reb-dev — COMMENTED** [review]
Looks good @lee00678! I left one comment, I think you'll need to rebase on main to address it

**Comment by lee00678** [comment]
> **CLA Assistant Lite bot:** 🎉 Thank you for your contribution. It appears you have not yet signed the F5 Contributor License Agreement (CLA), which is required for your changes to be incorporated into an F5 project. Please kindly read the [F5 CLA](https://github.com/f5/.github/blob/main/CLA/cla-markdown.md) and comment the following to agree:
> 
> I have hereby read the F5 CLA and agree to its terms
> 
> **Qian Liu** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).You can retrigger this bot by commenting **recheck** in this Pull Request

Agree

**Review by reb-dev — APPROVED** [review]
Thanks @lee00678! Merging.

**Pull request merged**